corehq/apps/app_manager/views/download.py

```diff
@@ -358,7 +358,7 @@ def download_media_profile(request, domain, app_id):
 @safe_cached_download
 def download_practice_user_restore(request, domain, app_id):
     if not request.app.copy_of:
-        make_async_build_v2.delay(request.app.get_id, request.app.domain, request.app.version)
+        make_build(request.app)
```

Comment:
`username` is a required arg, so I think this'll fail

Reply:
strange that this has always been like that, https://github.com/dimagi/commcare-hq/pull/24815/files#diff-8c8eb3bf8735832e5061795c93ce6e7fL361

Reply:
https://github.com/dimagi/commcare-hq/pull/24860/commits/9d9e328305719c7249f498383d26ea1717c10b68

Reply:
odd.  I guess that means it's not common to use a practice user on the latest saved app?